packages/use-dataloader/src/reducer.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable consistent-return */
```

Comment:
Shouldn't we have a default return ? Maybe returning the current state to avoid undefined behaviour ? Would also allow us to remove the eslint silencing :)

Reply:
No we can't. If we do that we can't have 100% test coverage (without importing the reducer himself)